CHANGELOG.md

```diff
@@ -51,6 +51,8 @@ Changes since v2.12
 - The OIDC attribute to use as the rems userid is now configurable via the `:oidc-userid-attribute`. See [docs/configuration.md](docs/configuration.md). (#2281)
 - The `:oidc-additional-authorization-parameters` config option. See [config-defaults.edn](resources/config-defaults.edn)
 - Applicants can now permanently delete drafts. (#2219)
+- It is now possible to optionally add info icon to text fields with collapsable description text (#1863)
```

Comment:
How about rephrasing this like: `All fields can have an info text, shown if the small icon is clicked. (#1863)`

Reply:
👍